tests/pytests/unit/modules/test_reg.py

```diff
@@ -14,19 +19,28 @@
 except ImportError:
     HAS_WIN32 = False
 
+
 pytestmark = [
-    pytest.mark.windows_whitelisted,
```

Comment:
Are you meaning to remove these markers?

Reply:
Unit tests always run on windows(unless explicitly skipped), contrary to integration tests.